src/api-ms-win-core-synch-l1-2-0/main.cpp

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+//
+// The code in this file was adapted from the STL on the 2021-07-05. Commit:
+// https://github.com/microsoft/STL/blob/e745bad3b1d05b5b19ec652d68abb37865ffa454/stl/src/atomic_wait.cpp
+//
+// It backports the following Windows 8 functions to Windows 7:
+// * WaitOnAddress
+// * WakeByAddressSingle
+// * WakeByAddressAll
+//
+
+#include <cstdint>
+#include <new>
+
+#include <winsdkver.h>
+#define _WIN32_WINNT 0x0601
+#include <sdkddkver.h>
+
+#define WIN32_LEAN_AND_MEAN
+#define NOMINMAX

```

Comment:
`NOMINMAX` should be included in `WIN32_LEAN_AND_MEAN`.

Reply:
Oh it's STL file, then nvm.

Reply:
As far as I understand it `WIN32_LEAN_AND_MEAN` doesn't include `NOMINMAX`.